Gordon360/Services/RecIM/SeriesService.cs

```diff
@@ -623,21 +645,48 @@ public async Task<EliminationRound> ScheduleElimRoundAsync(IEnumerable<SeriesTea
             var series = _context.Series.Find(involvedTeams.First().SeriesID);
 
             var teams = involvedTeams.Reverse();
-
+            var matches = new List<MatchViewModel>();
+            var byeTeams = new List<int>();
+            //bye logic
             while (!(((numTeams + numByes) != 0) && (((numTeams + numByes) & ((numTeams + numByes) - 1)) == 0))) //while not power of 2
             {
                 await UpdateSeriesTeamStats(new SeriesTeamPatchViewModel
                 {
                     ID = teams.Last().ID,
                     WinCount = 1 //Bye round
-                });
+                }) ;
+                byeTeams.Add(teams.Last().TeamID);
                 teams = teams.Take(--remainingTeamCount);
                 numByes++;
             }
 
-            var teamPairings = EliminationRoundTeamPairsAsync(teams);
-            var matches = new List<MatchViewModel>();
+            // logic for handling bye teams that play another buy team in the next round
```

Comment:
Two different spellings of `bye`?

Reply:
Misspelling from me, I was probably writing that at 3am whoops good catch!